qa/suites/fs/upgrade/featureful_client/upgraded_client/tasks/4-compat_client.yaml

```diff
@@ -6,7 +6,7 @@ tasks:
 - exec:
     mon.a:
       - ceph fs dump --format=json-pretty
-      - ceph fs required_client_features cephfs add metric_collect
+      - ceph fs required_client_features cephfs add dummy_test
```

Comment:
Looking at this more closely, do we really need a dummy bit? We can just have a reef/squid client masquerade as a client without an arbitrary bit, like `metric_collect`.

Reply:
(End of review: I think this is the best/simplest way forward.)

Reply:
I did think about that - the reason I kept the dummy feature bit is to allow for any future tinkering with a feature bit that is just there for enjoyment. Its isn't required for this change and I'm willing to rip it off.

Reply:
So, I guess we are moving forward with this as it is (dummy bit) and enhance the feature config interface later, yes?

Reply:
Updated withe dummy_test feature bit removed.